blob/fileblob/fileblob.go

```diff
@@ -30,25 +30,6 @@
 // (e.g., "~", which unescapes to "~", which escapes back to "%7E" != "~"),
 // aren't visible using fileblob.
 //
-// Open URLs
```

Comment:
Put part of this back? IMHO each driver package should have a `Open URLs` section (or maybe `OpenBucket URLs` now) that at least describe the scheme and a sample URL. It can refer to the `URLOpener` type for more details.